Gems/Multiplayer/Code/Include/Multiplayer/AutoGen/AutoComponent_Header.jinja

```diff
@@ -211,7 +211,11 @@ AZStd::array<{{ Property.attrib['Type'] }}, {{ Property.attrib['Count'] }}> m_{{
 {% elif Property.attrib['Container'] == 'Vector' %}
 AZStd::fixed_vector<{{ Property.attrib['Type'] }}, {{ Property.attrib['Count'] }}> m_{{ LowerFirst(Property.attrib['Name']) }};
 {% else %}
+  {% if Property.attrib['Init'] | length %}
+{{ Property.attrib['Type'] }} m_{{ LowerFirst(Property.attrib['Name']) }} = {{Property.attrib['Init']}};
+  {% else %}
```

Comment:
nit pick: indentation elsewhere in this file seems to keep the starting `{%` at the far left of the line while adding whitespace in front of the else/if to indicate nesting ([example](https://github.com/o3de/o3de/blob/development/Gems/Multiplayer/Code/Include/Multiplayer/AutoGen/AutoComponent_Header.jinja#L191-L201)). 
I'm not familiar enough with jinja to know if this is for syntactic reasons, but wanted to call it out as a deviation

Reply:
Great looking out! Fixed!